modules/collectd/manifests/plugin/docker.pp

```diff
@@ -2,7 +2,10 @@
 #
 # Collectd plugin for docker
 #
```

Comment:
You should put some documentation of the parameters here

Reply:
done